src/engine/source/builder/builders/stageParse.cpp

```diff
@@ -109,19 +106,9 @@ types::Lifter stageBuilderParse(const types::DocumentValue &def)
         parsers.emplace_back(newOp);
     }
 
-    try
-    {
-        auto check = std::get<types::CombinatorBuilder>(
-            Registry::getBuilder("combinator.broadcast"))(parsers);
-        return check;
-    }
-    catch (std::exception &e)
-    {
-        const char *msg = "Stage [parse] builder encountered exception "
-                          "chaining all mappings.";
-        LOG(ERROR) << msg << " From exception: " << e.what();
-        std::throw_with_nested(std::runtime_error(msg));
-    }
+    auto check = std::get<types::CombinatorBuilder>(
```

Comment:
On this **try** cleanup:
We are assuming the Registry was correctly initialized to not fail. I'm wondering if it is a good practice doing this.